pkg/api/meta/meta.go

```diff
@@ -321,16 +321,18 @@ func (resourceAccessor) SetResourceVersion(obj runtime.Object, version string) e
 // genericAccessor contains pointers to strings that can modify an arbitrary
 // struct and implements the Accessor interface.
 type genericAccessor struct {
-	namespace       *string
-	name            *string
-	generateName    *string
-	uid             *types.UID
-	apiVersion      *string
-	kind            *string
-	resourceVersion *string
-	selfLink        *string
-	labels          *map[string]string
-	annotations     *map[string]string
+	namespace         *string
```

Comment:
We can implement the interface on external objects too if need be.  I'm wondering if we need a way to verify who is using external objects and where (something written to glog.v(5) whenever someone uses a metadata accessor on an external type)?